README.md

```diff
@@ -72,11 +71,83 @@ steps in [contributing.md](contributing.md).
 
 Wait for approval of the Pull Request and merge the change.
 
-## Support
+### Code quality
+
+Code style guidelines ease understanding and maintaining code. Automated checks
+enforce some of the guidelines.
+
+#### Python
+
+*   Code must be formatted according to the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    according to the [yapf formatter](https://github.com/google/yapf).
+*   Code must not generate lint errors or warnings according to
+    [pylint](https://www.pylint.org/) configured for the
+    [Google Python Style Guide](https://google.github.io/styleguide/pyguide.html)
+    as specified in `.pylintrc`.
```

Comment:
make this a link to the checked in .pylintrc

Reply:
Done.